metabase/metabase
# Move timelines between collections

Epic https://github.com/metabase/metabase/issues/20289

How to test:
- Create multiple collections and a timeline within one of them
- Go to the timeline, open menu in the header and click `Move time`
- Select another collection and click Move
- Make sure the timeline is moved to another collection

<img width="653" alt="Screenshot 2022-04-11 at 12 30 57" src="https://user-images.githubusercontent.com/8542534/162711046-2923fe2f-3d46-4684-87bf-b4607a05a629.png">
<img width="653" alt="Screenshot 2022-04-11 at 12 30 48" src="https://user-images.githubusercontent.com/8542534/162711029-e28cf3fe-44f8-4a46-994a-0520a870f0d8.png">
